general/services/action.py

```diff
@@ -254,3 +254,29 @@ def create_submission_evaluation_action(submission):
             'status': submission.status,
         }
     )
+
+
+def course_registration_update_action(registration, user, data):
+    Action.create_action(
+        actor=user,
+        description='User update a course registration status.',
```

Comment:
Maybe change it to "User updated course registration status"

Reply:
Done.